### PR TITLE
Feature/admissions pdf

### PR DESCRIPTION
**Pull request by behzad1q**
keep admissions-pdf feature needs to be get updated with develop branch.

Actually, the purpose of this pull request is to review the code but not merge it to develop as we try to keep the feature updated.

when all good run 
``` 
docker build . -t uofa/s2i-shepherd-drupal:admissions-pdf
docker push uofa/s2i-shepherd-drupal:admissions-pdf
```

### CODE REVIEW

**Comment by pingers** [comment]
This will **not** be merged to develop. You need to maintain either:
1. a separate branch for wkhtmltopdf
2. a separate project for wkhtmltopdf.

The overhead is too large for wcms deployments.

File: Dockerfile [hunk 1]
```diff
@@ -54,6 +58,10 @@ RUN apt-get update \
   telnet \
   unzip \
   wget \
+&& echo 'deb http://apt.newrelic.com/debian/ newrelic non-free' | tee /etc/apt/sources.list.d/newrelic.list \
+&& wget -O- https://download.newrelic.com/548C16BF.gpg | apt-key add - \
+&& apt-get update \
+&& apt-get -y install newrelic-php5 \
```
Comment (pingers) [inline Dockerfile]: New Relic stuff is already in the Dockerfile - looks like a bad merge.

File: Dockerfile [hunk 2]
```diff
@@ -19,26 +19,28 @@ ENV LANG       en_AU.UTF-8
...
+&& apt-get -y --no-install-recommends install ca-certificates locales \
 && sed -i -e 's/# en_AU.UTF-8 UTF-8/en_AU.UTF-8 UTF-8/' /etc/locale.gen \
 && locale-gen en_AU.UTF-8 \
-&& apt-get -y dist-upgrade \
-&& apt-get -y install \
+&& apt-get -y upgrade \
+&& apt-get -y --no-install-recommends install \
   apache2 \
   bind9-host \
-  fontconfig \
```
Comment (singularo) [inline Dockerfile]: Aren't these needed for wkhtmltopdf?

**Comment by singularo** [comment]
We want to just close this PR, and keep the separate branch as its own thing, right?

**Pull request closed without merging**